test/apps/gups.c

```diff
@@ -412,12 +411,12 @@ SHMEMRandomAccess(void)
 
   if (MyProc == 0) {
     fprintf( outFile, "Running on %d processors\n", NumProcs);
-    fprintf( outFile, "Total Main table size = 2^" FSTR64 " = " FSTR64 " words\n",
+    fprintf( outFile, "Total Main table size = 2^%" PRIu64 " = %" PRIu64 " words\n",
```

Comment:
This is true, and I'm generally fine with accepting C99 as a baseline for SOS.  In fairness to the PR, the C99 dependence was already there because the fixed-size integer types were added in '99.